src/rcore.c

```diff
@@ -1534,6 +1534,16 @@ void SetWindowSize(int width, int height)
 #endif
 }
 
+// Set window opacity, value opacity is between 0.0 and 1.0
+void SetWindowOpacity (float opacity)
```

Comment:
It seems there is an extra space before parenthesis, please review it.

Reply:
I'm really sorry, it was intentional I thought I read it in CONTRIBUTING.md but anyways I'll fix it.